src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -281,11 +281,11 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 				}
 				rows.push(row);
 				if (!isString(topProp.value) && !isString(bottomProp.value)) {
-					this.convertPropertiesToTableRows(topProp.value, bottomProp.value, rows.length - 1, indent + 2, rows);
+					row['treeGridChildren'] = this.convertPropertiesToTableRows(topProp.value, bottomProp.value);
 				} else if (isString(topProp?.value) && !isString(bottomProp.value)) {
-					this.convertPropertiesToTableRows(undefined, bottomProp.value, rows.length - 1, indent + 2, rows);
+					row['treeGridChildren'] = this.convertPropertiesToTableRows(undefined, bottomProp.value);
 				} else if (!isString(topProp.value) && !isString(bottomProp.value)) {
```

Comment:
This `else if` condition is the same as the first `if` statement. Is that intentional?